test/services/ScratchServiceSpec.scala

```diff
@@ -52,9 +52,9 @@ class GuidanceServiceSpec extends BaseSpec {
 
       MockGuidanceConnector
         .submitScratchProcess(dummyProcess)
-        .returns(Future.successful(Right(SaveScratchSubmissionResponse(uuid))))
+        .returns(Future.successful(Right(ScratchResponse(uuid))))
```

Comment:
The test description could do with updating.

Reply:
Updated in latest commit